docs/user-guide/install-zowe-smpe.md

```diff
@@ -863,7 +882,11 @@ In this step, you run the sample job ZWE7APLY to apply Zowe. This step can take
 
 2. After you take actions that are indicated by the APPLY CHECK, remove the CHECK operand and run the job again to perform the APPLY.
 
-**Note:** The GROUPEXTENDED operand indicates the SMP/E applies all requisite SYSMODs. The requisite SYSMODS might be applicable to other functions.
+**Notes:** 
+- The GROUPEXTENDED operand indicates the SMP/E applies all requisite SYSMODs. The requisite SYSMODS might be applicable to other functions.
+- If you are installing Zowe API Mediation Layer, you need to follow the preceding steps to run the jobs ZWE7APLY and ZWE8ACPT twice. 
```

Comment:
Is this specific to APIML installation or is it a global requirement?

Reply:
@nannanli It is not specific to APIML installation. APIML installation is included in Zowe SMP/E installation. We run ZWE7APLY and ZWE8ACPT once when we install Zowe SMP/E base FMID. I would say, above statement is not correct.